docs/content/preview/explore/transactions/distributed-transactions-ysql.md

```diff
@@ -201,7 +209,7 @@ where transaction_mode is one of:
   [ NOT ] DEFERRABLE
 ```
 
```

Comment:
In the "Run a transaction" section:

_After the above transaction succeeds, the table should look as follows:_   and then there's a SELECT statement. We also need to get rid of "above"